api-v3-beta.rst

```diff
@@ -0,0 +1,62 @@
+.. post:: August 8, 2019
+   :tags: api, feature, reference
+   :author: Manuel
+   :location: BCN
+
+.. meta::
+   :description lang=en:
+
+      Announcing API v3 Beta as invite-only status. Help us to make it *even* better!
+
+========================
+ Announcing API v3 Beta
+========================
+
+In the last months, we have been working on making our API better.
+Considering the limitations of our current REST API v2,
+we decided to make a bigger step forward and create our new shiny API v3 with better and bigger features,
+putting the focus on the user cases we heard about from existing users.
```

Comment:
use cases?

Reply:
Hrm... Interesting. Eric suggested to change this to `user cases` in https://github.com/readthedocs/blog/pull/75#discussion_r312178396

I thought it was "use cases", but... you may know better :)